Plugin/Quote/Model/AbstractCheckoutTrigger.php

```diff
@@ -0,0 +1,71 @@
+<?php
+
+namespace Yotpo\SmsBump\Plugin\Quote\Model;
+
+use Yotpo\SmsBump\Model\Sync\Checkout\Processor as CheckoutProcessor;
+use Magento\Framework\Registry;
+use Magento\Quote\Api\CartRepositoryInterface;
+use Yotpo\SmsBump\Model\Config;
+use Magento\Quote\Model\Quote as QuoteModel;
```

Comment:
I think we should remove the ModelQuote reference, and keep it as Quote, as it a bit confusing, what do you think?